### DISCUSSION TITLE
Problem with Airflow

### DISCUSSION THREAD

**Discussion by ShlomiA7** [apache/airflow#36089] [General]
how can i slove this problem ? 
i cant see the logs 
![Airflow prob](https://github.com/apache/airflow/assets/114301932/f7a39ce5-22ec-4b29-994f-776830c2e9f4)

**Comment by Taragolis** [comment]
**Option 1**: Please Contact Server admin

**Option 2**: If you are server admin then have a look logs on your webserver. And provide additional information here or try to google by key error, most possible someone already have this problem.

**Option 3**: If you are server admin you also might enable traceback into the webserver, it disabled by default by the security reason, by set [`[webserver] expose_stacktrace`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#expose-stacktrace) to `True`. And provide additional information here or try to google by key error, most possible someone already have this problem.
